vms/platformvm/validators/manager.go

```diff
@@ -247,7 +246,11 @@ func (m *manager) makePrimaryNetworkValidatorSet(
 		return nil, 0, err
 	}
 	if currentHeight < targetHeight {
-		return nil, 0, database.ErrNotFound
+		return nil, 0, fmt.Errorf("failed to fetch validator set (requested P-Chain Height: %d, SubnetID: %s, current P-chain Height: %d)",
+			targetHeight,
+			constants.PrimaryNetworkID,
+			currentHeight,
+		)
```

Comment:
we use this error in one of the clients of this validator manager. Better make it available to all clients